Senpa.io/dist/metadata.json

```diff
@@ -9,7 +9,7 @@
         "fr": "Le meilleur serveur privé io Dual Agario Unblocked."
     },
     "service": "Senpa.io",
-    "version": "1.0",
+    "version": "1.3",

```

Comment:
it doesnt make sence that it goes from 1.0 to 1.3 at once...
Use: MAJOR,MINOR,PATCH